### DISCUSSION TITLE
Change default directory when exporting data as text

### DISCUSSION THREAD

**Discussion by mahadea** [dbeaver/dbeaver#16134] [DBeaver Development]
Hi
Running DBeaver 22.0.2.202204041542 on Win 10.
Is it possible to change the default directory setting on this window from c:\users\mahadea to something else, so that I don't need to keep changing it every time I am exporting data?

Thanks
![image](https://user-images.githubusercontent.com/9688051/162479822-e2649e81-0ccd-43f0-a916-9d12e962f1c3.png)

**Comment by hope-courage** [comment]
Hello, I want to set the Export Data button as not optional. Which module (the file of java) should I view?
